eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,116 @@
+$Language = "dotnet"
+$PackageRepository = "Nuget"
```

Comment:
Same comment about PackageRepository if we don't need it we should just exclude it and use the language.